Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -200,6 +200,52 @@ class ArgumentUtility
     ArgumentUtility.CheckNotNull(name, value);
     ArgumentUtility.CheckTypeIsJQueryObject(name, value);
   };
+
+  // Resolves any provided css selector, checks the element for null and returns it.
+  public static ResolveCssSelectorOrElement<TElement extends Element> (selectorOrElement: CssSelectorOrElement<TElement>, context?: ParentNode): TElement
+  {
+    if (TypeUtility.IsString(selectorOrElement))
+    {
+      const queryContext = context ? context : document;
+      const resolvedElement = queryContext.querySelector(selectorOrElement) as TElement;
+      if (!resolvedElement)
+        throw ('Error: Cannot find an element specified by selector "' + selectorOrElement + '".');
+
+      return resolvedElement;
+    }
+    else if (TypeUtility.IsObject(selectorOrElement))
+    {
+      ArgumentUtility.CheckNotNull("selectorOrElement", selectorOrElement);
+      return selectorOrElement;
+    }
+    else
+    {
+      throw ('Error: The type of parameter "selectorOrElements" is "' + (typeof selectorOrElement) + '" but "string" or "object" was expected.');
+    }
+  }
+
+  // Resolves any provided css selector, and returns the found elements as an array.
+  public static ResolveCssSelectorOrElements<TElement extends Element> (selectorOrElements: CssSelectorOrElements<TElement>, context?: ParentNode): TElement[]
```

Comment:
Question (also affects the naming issue from above): Do we a) actually have a case where we want to pass a list of elements instead of a css selector? Maybe the either/or variant is only for single-valued results.

Reply:
I am not yet sure if we have a case where multiple elements are selected. Most occurrences will be single-element only but there might be some that do have multiple elements. Therefore, I would add it for now and remove it when it is not used after jQuery has been removed

Reply:
okay. So basically this oculd mean we do a global update after we 're done to cleanup and use the "right" name for what's left over?

Reply:
Yes we can do that. And if the multiple element resolver method is not used we can remove it then.